debian/rules

```diff
@@ -5,16 +5,19 @@ include /usr/share/dpkg/pkg-info.mk
 FLAKE8 := $(shell flake8 --version 2> /dev/null)
 
 include /etc/os-release
+# see https://bugs.launchpad.net/ubuntu/+source/ubuntu-advantage-tools/+bug/1840091/comments/3
 ifeq (${VERSION_ID},"14.04")
-LIBAPT_PKG_DEP="libapt-pkg4.12 (>= 1.0.1ubuntu2.23)"
+APT_PKG_DEPS="apt (>= 1.0.1ubuntu2.23), apt-transport-https (>= 1.0.1ubuntu2.23), apt-utils (>= 1.0.1ubuntu2.23), libapt-inst1.5 (>= 1.0.1ubuntu2.23), libapt-pkg4.12 (>= 1.0.1ubuntu2.23) "
 else ifeq (${VERSION_ID},"16.04")
-LIBAPT_PKG_DEP="libapt-pkg5.0 (>= 1.2.31)"
+APT_PKG_DEPS="apt (>= 1.2.32), apt-transport-https (>= 1.2.32), apt-utils (>= 1.2.32), libapt-inst2.0 (>= 1.2.32), libapt-pkg5.0 (>= 1.2.32)"
 else ifeq (${VERSION_ID},"18.04")
-LIBAPT_PKG_DEP="libapt-pkg5.0 (>= 1.6.9)"
+APT_PKG_DEPS="apt (>= 1.6.11), apt-utils (>= 1.6.11), libapt-inst2.0 (>= 1.6.11), libapt-pkg5.0 (>= 1.6.11)"
```

Comment:
As explained in the description, I grabbed versions which had this changelog entry: `  * Add test case for local-only packages pinned to never`. They are also the ones current at the time the comment https://bugs.launchpad.net/ubuntu/+source/ubuntu-advantage-tools/+bug/1840091/comments/3 was written.
I don't mind grabbing the latest and greatest available in each ubuntu release, and, in fact, this is what will happen in practice, since updates get overwritten.

Reply:
@panlinux Josh mentioned we should just leave the old versions as is so we know the time frame we originally set those min version numbers (and as you said, -updates will update to the latest anyway).

Reply:
yep we'll ignore those version updates. As you said, they'll be pulled anyway.